src/box/lua/config/descriptions.lua

```diff
@@ -2016,6 +2016,12 @@ I['replication.timeout'] = format_text([[
     tries to reconnect to the master.
 ]])
 
+
+I['replication.reconnect_timeout'] = format_text([[
+    The timeout (in seconds) between attempts to reconnect to a master
+    in case of connection failure.
+]])
```

Comment:
I would add an information about the default value.

Reply:
Done

Reply:
Seems like this piece of the documentation is not changed. Forgot to push or does we misunderstood each other?

Reply:
BTW, from the documentation request:

> then it equals to reconnect_timeout

Seems like `replication_timeout` is meant.

Reply:
Now it's definitely fixed, my bad) The doc request is also fixed